source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown

```diff
@@ -0,0 +1,122 @@
+install `solidus` without `solidus_frontend`, we'll need to manually
+add the rest of the Solidus components individually:
+
+```sh
+rails new store --skip-javascript
+cd store
+bundle add solidus_core solidus_backend solidus_api solidus_sample
+bin/rails generate solidus:install --auto-accept
+```
+
+Once done, all we need to do to install `solidus_starter_frontend` is to run
+the following line:
+
+```sh
+LOCATION="https://raw.githubusercontent.com/solidusio/solidus_starter_frontend/master/template.rb" \
+  bin/rails app:template
+```
+
+## What can I expect to see once the frontend is installed?
+
+<div class="carousel">
+  <div class="carousel-list">
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/10-home.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Home page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/20-product.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Product page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/30-cart.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Cart page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/40-login.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Login page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/50-checkout-address.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Address page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/60-checkout-delivery.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Delivery page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/70-checkout-payment.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Payment page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/80-checkout-confirm.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Confirm page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/90-checkout-complete.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Complete page</figcaption>
+    </figure>
+  </div>
+</div>
```

Comment:
@kennyadsl We can customize this carousel to better present the screenshots. Let me know which format in https://kenwheeler.github.io/slick/#demos is ideal.